artichoke-backend/src/eval.rs

```diff
@@ -220,4 +229,15 @@ mod tests {
         let err = interp.eval(b"require 'fail'").unwrap_err();
         assert_eq!("SyntaxError", err.name().as_ref());
     }
+
+    #[test]
+    fn eval_file_error_file_not_found() {
+        let mut interp = interpreter();
+        let err = interp.eval_file(Path::new("no/such/file.rb")).unwrap_err();
```

Comment:
this is a really nifty way to test this. I thought we'd need a UI test. Thank you!

Reply:
I was just copying `interp.eval()` from other test cases 😄